importer/api/__init__.py

```diff
@@ -130,9 +132,9 @@ def import_children(course, element, parent):
     of an XML tree.
 
     Args:
-        course (learningresources.Course): Course
+        course (Course): Course
         element (lxml.etree): XML element within xbundle
-        parent (learningresources.LearningResource): Parent LearningResource
+        parent (LearningResource): Parent LearningResource
```

Comment:
Same here, for sphinx, should be `learningresources.models.LearningResource` to properly link in the api docs (I think anyway)